index.html

```diff
@@ -72,7 +72,7 @@
     </div>
 
     <div class='bar floating top' id='top-bar' hide=false>
-      <div id="volume-slider-box">
+      <div id="volume-slider-box" class="tooltip-1  tooltip" data-tooltip="Volume">
```

Comment:
Had to check the screenshot for figuring out the diff between `tooltip` & `tooltip-1`– what if we call them `tooltip` and `tooltip-e` (tooltip east)?

Reply:
It is done and renamed to **tooltip-e**